tests/test_analysis.py

```diff
@@ -155,5 +155,25 @@ def test_analysis(self):
         subprocess.check_call(mpiCmd + cmd)
 
 
+class TestIntersect(unittest.TestCase):
+    def setUp(self):
+        os.chdir(os.path.join(tutorialDir, "intersection"))
+
+    def test_pyhyp(self):
+        cmd = ["./get-input-files.sh"]
+        subprocess.check_call(cmd)
```

Comment:
`./get-input-files.sh` does not seem to be returning any codes to check. A bit dirty, but a quick way would be to add `set -e` at the top of the script to exit with some non-zero code without controlling.